damus/Views/Wallet/TransactionsView.swift

```diff
@@ -46,15 +47,15 @@ struct TransactionView: View {
                         .bold()
                         .foregroundColor(DamusColors.adaptableBlack)
                     
-                    Text("\(relativeDate)")
```

Comment:
No need to interpolate the string. It's already a string. This also avoids the translation exporter tool from pulling this string out for translation.